core/src/main/java/com/datastax/oss/driver/api/core/config/DefaultDriverOption.java

```diff
@@ -808,6 +807,14 @@
    * <p>Value-type: int
    */
   SESSION_LEAK_THRESHOLD("advanced.session-leak.threshold"),
+  /**
+   * The time after the node level metrics will be evicted. The eviction will happen only if none of
+   * the enabled node-level metrics is updated for a given node for a given time. When the
+   * eviction-time will elapse, all metrics for a given node are removed.
+   *
+   * <p>Value-type: {@link java.time.Duration Duration}
+   */
+  METRICS_NODE_EVICTION_TIME("advanced.metrics.node.eviction-time"),
```

Comment:
```suggestion
  METRICS_NODE_EXPIRE_AFTER("advanced.metrics.node.expire-after"),
```

I don't like much the usage of the word "time" in the sense of "expiration interval". Also "eviction" is a cache vocabulary, but users don't need to know we are using a cache with eviction based on activity, this is an implementation detail.